eos/rare-b-decays/lambda-b-to-lambda-nu-nu.cc

```diff
@@ -0,0 +1,178 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2023 Méril Reboud
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <eos/rare-b-decays/lambda-b-to-lambda-nu-nu.hh>
+#include <eos/form-factors/form-factors.hh>
+#include <eos/maths/integrate.hh>
+#include <eos/maths/power-of.hh>
+#include <eos/models/model.hh>
+#include <eos/utils/destringify.hh>
+#include <eos/utils/kinematic.hh>
+#include <eos/utils/options.hh>
+#include <eos/utils/private_implementation_pattern-impl.hh>
+
+namespace eos
+{
+    template <>
+    struct Implementation<LambdaBToLambdaDineutrino>
+    {
+        std::shared_ptr<Model> model;
+
+        std::shared_ptr<FormFactors<OneHalfPlusToOneHalfPlus>> form_factors;
+
+        Parameters parameters;
+
+        UsedParameter m_Lambda_b;
+
+        UsedParameter tau_Lambda_b;
+
+        UsedParameter m_Lambda;
+
+        UsedParameter g_fermi;
+
+        UsedParameter alpha_e;
+
+        UsedParameter hbar;
+
+        UsedParameter mu;
+
+        static const std::vector<OptionSpecification> options;
+
+        std::function<complex<double> ()> lambda_t;
+        std::function<WilsonCoefficients<wc::SBNuNu> ()> wc;
+
+        GSL::QAGS::Config int_config;
+
+        bool cp_conjugate;
+
+        Implementation(const Parameters & p, const Options & o, ParameterUser & u) :
+            model(Model::make(o.get("model", "SM"), p, o)),
+            parameters(p),
+            m_Lambda_b(p["mass::Lambda_b"], u),
+            tau_Lambda_b(p["life_time::Lambda_b"], u),
+<<<<<<< Updated upstream
```

Comment:
Oops.

Reply:
hmm weird, looks like a rebase problem, because I did not merged anything